dev/bots/test.dart

```diff
@@ -75,11 +75,18 @@ Future<Null> _verifyInternationalizations() async {
 
   final String localizationsFile = path.join('packages', 'flutter_localizations', 'lib', 'src', 'l10n', 'localizations.dart');
 
-  final EvalResult sourceContents = await _evalCommand(
-    'powershell',
-    <String>['\$PSDefaultParameterValues["*:Encoding"]="utf8";(gc $localizationsFile) -join "`n"'],
-    workingDirectory: flutterRoot,
-  );
+  final EvalResult sourceContents =
```

Comment:
nit: Since each branch is multiple lines I'd almost prefer a real if/else to make it easier to read...